lms/static/js/student_profile/views/share_modal_view.js

```diff
@@ -0,0 +1,48 @@
+;(function (define, undefined) {
+    'use strict';
+    define(['gettext', 'jquery', 'underscore', 'backbone', 'moment',
+            'text!templates/student_profile/share_modal.underscore'],
+        function (gettext, $, _, Backbone, Moment, badgeModalTemplate) {
+
+            var ShareModalView = Backbone.View.extend({
```

Comment:
Each view class should have a corresponding spec file that tests its behavior. Please add tests for the keyboard handling, and the rendering (especially since it looks like that part is broken since the update to Underscore).